docs/find_and_replace_action_json_patch.md

```diff
@@ -0,0 +1,595 @@
+{"op": "move","from": "/temp_description_array","path": "/description"},
+```
+
+This is a multi step operation. Given that JSON Patch can not "cast" types and depends on a given datatype to be present before, e.g. adding a new value to it, we use here a temporary key.
+Notice that you can not “add“ or “move“ to e.g. the position `0` because the destination array is indeed empty (that will fail). But by using the `dash` you can command it to put it at the end, which on an empty list is also at the beginning (we are starting to understand this!). 
+
+3. And the extra LOD entry for wikidata, in this case the Q Item for [collie](https://www.wikidata.org/wiki/Q1196071) (type of herding dog)
+
+```JSON
+{ "op": "add", "path": "/subject_wikidata/-", "value": {
+        "uri": "https://www.wikidata.org/wiki/Q1196071",
+        "label": "collie"
+    }
+}
+```
+
+4. Finally we are going to copy the `geographic_location.state`  value and put it into `subjects_local`:
+
+```JSON
+{"op": "remove", "path": "/subjects_local"},
+{"op": "add", "path": "/subjects_local", "value": []},
+{"op": "copy", "from": "/geographic_location/state", "path": "/subjects_local/-"}
+```
+
+Why so many operations? Because initially ` "subjects_local"` had a value of `null` and it is not suited to generate/add to it as a multivalued key because of that.  So we need to remove it first, recreate it as an empty list and then we can copy. **Pro Note:** you could partially rewrite this as  `replace` operation!
+
+??? info "what if it subjects_local has data?
+
+	You will lose it! you can add a “test“ or move data instead of recreating. Many choices.
+
+The final JSON Patch will look like this. Copy it into the Configuration JSON Patch commands form field: 
+
+```JSON
+[
+    {"op": "test", "path": "/type", "value": "Photograph"},
+    {"op": "test", "path": "/ismemberof", "value": [16]},
+    {"op": "add","path": "/temp_description_array","value": []},
+    {"op": "move","from": "/description","path": "/temp_description_array/-"},
+    {"op": "move","from": "/temp_description_array","path": "/description"},
+    {"op": "add","path": "/subject_wikidata/-","value": 
+        {
+        "uri": "https://www.wikidata.org/wiki/Q1196071",
+         "label": "collie"
+         }
+    },
+    {"op": "remove", "path": "/subjects_local"},
+    {"op": "add", "path": "/subjects_local", "value": []},
+    {"op": "copy", "from": "/geographic_location/state", "path": "/subjects_local/-"}
+]
+```
+
+??? note "The inverse process online"
+
+	Now that you know (or are starting to understand) the manual process you can also try this [online tool](https://json-patch-builder-online.github.io) that allows you, based on a source and a destination JSON, generate the needed JSON Patch to mutate one JSON into the another. The logic might not be always what you need and most likely it will not take in account that you actually need to move values and will prefer to fix values to be added via an add operation
```

Comment:
```suggestion
    Now that you know (or are starting to understand) the manual process you can also try this [online tool](https://json-patch-builder-online.github.io) that allows you, based on a source and a destination JSON, generate the needed JSON Patch to mutate one JSON into the another. The logic might not be always what you need and most likely it will not take in account that you actually need to move values and will prefer to fix values to be added via an add operation
```